src/Magnum/Platform/GlfwApplication.h

```diff
@@ -0,0 +1,753 @@
+            R = 'r',                    /**< Letter R */
+            S = 's',                    /**< Letter S */
+            T = 't',                    /**< Letter T */
+            U = 'u',                    /**< Letter U */
+            V = 'v',                    /**< Letter V */
+            W = 'w',                    /**< Letter W */
+            X = 'x',                    /**< Letter X */
+            Y = 'y',                    /**< Letter Y */
+            Z = 'z',                    /**< Letter Z */
+
+            /* Function keys */
+            Esc = GLFW_KEY_ESCAPE,              /**< Escape */
+            Enter = GLFW_KEY_ENTER,             /**< Enter */
+            Tab = GLFW_KEY_TAB,                 /**< Tab */
+            Backspace = GLFW_KEY_BACKSPACE,     /**< Backspace */
+            Insert = GLFW_KEY_INSERT,           /**< Insert */
+            Delete = GLFW_KEY_DELETE,           /**< Delete */
+            CapsLock = GLFW_KEY_CAPS_LOCK,      /**< Caps lock */
+            ScrollLock = GLFW_KEY_SCROLL_LOCK,  /**< Scroll lock */
+            NumLock = GLFW_KEY_NUM_LOCK,        /**< Num lock */
+            PrintScreen = GLFW_KEY_PRINT_SCREEN,/**< Print screen */
+            Pause = GLFW_KEY_PAUSE,             /**< Pause */
+            Numpad0 = GLFW_KEY_KP_0,            /**< Numpad 0 */
+            Numpad1 = GLFW_KEY_KP_1,            /**< Numpad 1 */
+            Numpad2 = GLFW_KEY_KP_2,            /**< Numpad 2 */
+            Numpad3 = GLFW_KEY_KP_3,            /**< Numpad 3 */
+            Numpad4 = GLFW_KEY_KP_4,            /**< Numpad 4 */
+            Numpad5 = GLFW_KEY_KP_5,            /**< Numpad 5 */
+            Numpad6 = GLFW_KEY_KP_6,            /**< Numpad 6 */
+            Numpad7 = GLFW_KEY_KP_7,            /**< Numpad 7 */
+            Numpad8 = GLFW_KEY_KP_8,            /**< Numpad 8 */
+            Numpad9 = GLFW_KEY_KP_9,            /**< Numpad 9 */
+            NumpadDecimal = GLFW_KEY_KP_DECIMAL,    /**< Numpad Decimal */
+            NumpadDivide = GLFW_KEY_KP_DIVIDE,      /**< Numpad Divide */
+            NumpadMultiply = GLFW_KEY_KP_MULTIPLY,  /**< Numpad Multiply */
+            NumpadSubtract = GLFW_KEY_KP_SUBTRACT,  /**< Numpad Subtract */
+            NumpadAdd = GLFW_KEY_KP_ADD,        /**< Numpad Add */
+            NumpadEnter = GLFW_KEY_KP_ENTER,    /**< Numpad Enter */
+            NumPadEqual = GLFW_KEY_KP_EQUAL,    /**< Numpad Equal */
+            LeftShift = GLFW_KEY_LEFT_SHIFT,    /**< Left shift */
+            LeftCtrl = GLFW_KEY_LEFT_CONTROL,   /**< Left control */
+            LeftAlt = GLFW_KEY_LEFT_ALT,        /**< Left alt */
+            LeftSuper = GLFW_KEY_LEFT_SUPER,    /**< Left super */
+            RightShift = GLFW_KEY_RIGHT_SHIFT,  /**< Right shift */
+            RightCtrl = GLFW_KEY_RIGHT_CONTROL, /**< Right control */
+            RightAlt = GLFW_KEY_RIGHT_ALT,      /**< Right alt */
+            RightSuper = GLFW_KEY_RIGHT_SUPER,  /**< Right super */
```

Comment:
Yes. Seems to be way easier than with SDL -- they're provided directly in the last parameter in the key event callback.